assets/scss/modules/_template-service-info.scss

```diff
@@ -8,8 +8,22 @@
     margin: 0 auto;
   }
 
-  .logo {
-      margin-top: 10px;
+  .logo,
+  .last-login {
+    margin-top: em(10);
+  }
+
+  .last-login {
+    margin-bottom:em(10);
+  }
+
+  .alert--info {
+    position:relative;
+    margin:em(10) 0 em(10) 0;
+  }
+
+  .more-details {
```

Comment:
Realised this should probably be `.last-login__details` to follow the BEM(ish) convention :)